src/test/end-to-end/shared/testingSupportApi.ts

```diff
@@ -34,28 +34,82 @@ export const deleteLocation = async (locationId: string) => {
     try {
         await superagent
             .delete(`${testConfig.DATA_MANAGEMENT_BASE_URL}/locations/${locationId}`)
-            .set('x-provenance-user-id', 'get it from environment variable')
-            .set({ Authorization: 'Bearer ' + token.access_token });
+            .set('x-provenance-user-id', `${testConfig.SYSTEM_ADMIN_PROVENANCE_ID}`)
+            .set({Authorization: 'Bearer ' + token.access_token});
     } catch (e) {
         throw new Error(`Failed to delete location with locationId: ${locationId}, http-status: ${e.response?.status}`);
     }
 };
 
-export const createSubscription = async (locationId: string) => {
+export const createSubscription = async (locationId: string, locationName: string, userId: string) => {
     const payload = {
         channel: 'EMAIL',
-        searchType: '100',
+        searchType: 'LOCATION_ID',
         searchValue: locationId,
-        locationName: 'Test Court',
+        locationName: locationName,
         listType: null,
-        userId: '12',
+        userId: userId,
     };
     try {
         await superagent
-            .post('http://localhost:4550/subscription')
+            .post(`${testConfig.SUBSCRIPTION_MANAGEMENT_BASE_URL}/subscription`)
             .send(payload)
-            .set('x-user-id', 'get it from environment variable');
+            .set('x-user-id', `${testConfig.TEST_USER_ID}`);
     } catch (e) {
-        throw new Error(`Failed : ${locationId}, http-status: ${e.response?.status}`);
+        throw new Error(`Create subscription failed for: ${locationName}, http-status: ${e.response?.status}`);
     }
 };
+
+export const deleteSubscription = async (userId: string) => {
+    try {
+        await superagent
+            .delete(`${testConfig.SUBSCRIPTION_MANAGEMENT_BASE_URL}/subscription/user/${userId}`)
+            .set('x-user-id', `${testConfig.TEST_USER_ID}`);
```

Comment:
Also need a bearer token on this one too :)

Reply:
Added token :)